external/model-preparation-algorithm/configs/detection/cspdarknet_yolox_cls_incr/template.yaml

```diff
@@ -0,0 +1,51 @@
+# Description.
```

Comment:
First, erase "template.yaml" and leave "template_experimental.yam'". The name change will be carried out in another PR.

Reply:
Please apply this to all model templates the same.